_posts/0000-01-02-Marcia201.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
```

Comment:
Thanks but I am very confused on what I'm doing I will need to go back and read over it so that I understand

Reply:
> 👍🏽

Dfgf

Reply:
> 👍🏽

Dfgf